### ISSUE TITLE
Support for ClickHouse (or other dbt warehouses)

### ISSUE THREAD

**Issue by akurdyukov** [lightdash/lightdash#8890]
### Description

I'd like to use lightdash with clickhouse. If there any specific reasons why only verified adapters are supported?

### What problem does this solve?

ClickHouse as column-based database with rich amount of analytical functions is pretty convenient as a warehouse.

**Comment by TuringLovesDeathMetal** [comment]
👋 Hey @akurdyukov 

we only support dbt verified adapters with Lightdash: https://docs.getdbt.com/docs/verified-adapters

Currently, clickhouse isn't on this list. Once it is, we'll add it as an option in Lightdash!

We also support Trino, which you can use as a way to get Clickhouse working in Lightdash. You can read more about this here: https://docs.lightdash.com/get-started/setup-lightdash/connect-project#trino

I'm going to close this issue as not planned becuase we won't support clickhouse until it's a verified dbt adapter!

**Comment by TuringLovesDeathMetal** [comment]
^btw, the reason for only supporting verified adapters is because they have requirements for maintenance that other adapters don't (e.g. they need to upgrade to support the latest dbt version within xxx time). 

We got a bit burned by this a while ago (one of the adapters didn't upgrade, so we couldn't upgrade _**any**_ adapters). So we made the decision to only support dbt verified adapters after that!
